src/diffusers/image_processor.py

```diff
@@ -164,6 +164,20 @@ def postprocess(
         image,
         output_type: str = "pil",
     ):
+        if output_type not in ["latent", "pt", "np", "pil"]:
+            deprecation_message = (
+                f"the output_type {output_type} is outdated. Please make sure to set it to one of these instead: "
+                "`pil`, `np`, `pt`, `latent`"
+            )
+            deprecate("Unsupported output_type", "1.0.0", deprecation_message, standard_warn=False)
+            output_type = "np"
+
+        if output_type == "latent":
+            return image
+
+        if image.min() < 0:
+            image = (image / 2 + 0.5).clamp(0, 1)
```

Comment:
oh so this is just to make sure it won't denormalize anything twice (like the question you asked below) - it will always denormalize unless it's already within `[0, 1]`

Reply:
Good catch here @pcuenca - actually I think we should indeed have this default to `True` always since we currently always doing this normalization as a post-processing: https://github.com/huggingface/diffusers/blob/80bc0c0ced1566549dec606f5069e909b86e86b0/src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py#L436 


@yiyixuxu We should do this check for the input, not the output